lib/dialects/postgres/query-generator.js

```diff
@@ -417,8 +417,13 @@ var QueryGenerator = {
 
   addLimitAndOffset: function(options) {
     var fragment = '';
-    if (options.limit) fragment += ' LIMIT ' + this.escape(options.limit);
-    if (options.offset) fragment += ' OFFSET ' + this.escape(options.offset);
+
+    if (typeof options.limit !== "undefined" && options.limit !== null) {
```

Comment:
No need for the double negator in that case, but yeah might be slightly cleaner :)
Could also simply check `typeof options.limit === 'number'`